spec/controllers/questions_controller_spec.rb

```diff
@@ -45,5 +45,121 @@
     end
   end
 
+  describe "GET #edit" do
+    let(:question) { FactoryGirl.create(:question) }
 
+    context "ログインしていない場合" do
+      it "編集画面が見れないこと" do
+        get :edit, {id: question.id}
+        expect(assigns(:question)).not_to eq question
+      end
+
+      it "編集画面ではなくサインイン画面にリダイレクトされること" do
+        get :edit, {id: question.id}
+        expect(response).to redirect_to('/users/sign_in')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.